### DISCUSSION TITLE
VS code only wsl and not wsl-ubuntu

### DISCUSSION THREAD

**Discussion by vhamilton-caseopp** [Cyfrin/foundry-full-course-cu#336] [Q&A] [answered]
Hi, I'm trying to get WSL installed and set up for vs code. I have went through the setup but it's only showing wsl and not wsl-ubuntu. I have ubuntu installed and from the ubuntu window in terminal open vs code 

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/113548746/3dbd3d33-63df-4bc3-ab94-8eded3628292)

**Comment by cromewar** [comment, marked as answer]
Hello @vhamilton-caseopp that's ok, WSL have a core running on ubuntu, and then you can install specific distributions so it will work fine.
Also you can run vscode from the WSL console using `code .` so it will run on the current distro you are working with on WSL.
